code/modules/surgery/organs/heart.dm

```diff
@@ -142,3 +143,16 @@
 /datum/client_colour/cursed_heart_blood
 	priority = 100 //it's an indicator you're dieing, so it's very high priority
 	colour = "red"
+
+/obj/item/organ/heart/cybernetic
+	name = "cybernetic heart"
+	desc = "An electronic device designed to mimic the functions of an organic human heart. Offers no benefit over an organic heart other than being easy to make."
+	icon_state = "heart-c"
+	origin_tech = "biotech=5"
```

Comment:
when i made this comment i had the lowest common denominator of scientist skill in mind
yes i see people not able to get past 4
yes it pains me